server/embedded/README.md

```diff
@@ -0,0 +1,36 @@
+# embedded
+The LabKey server application can now be started with embedded tomcat.
+
+#### Setup
+1. Uncomment useEmbeddedTomcat, useLocalBuild and useSsl (if you have the local ssl setup, more instructions below) in root level gradle.propeties file
+2. gradle refresh
+3. pickpg/pickmssql
+4. gradle cleanBuild, cleanDeploy and deployApp
+5. Select LabKeyEmbbedded_Dev from IntelliJ run configurations
```

Comment:
```suggestion
### Setup
1. Within the root-level `gradle.properties` file, uncomment the `useEmbeddedTomcat` and `useLocalBuild` properties.  If using SSL, also uncomment the `useSsl` property (and see further instructions below).
2. `gradlew cleanBuild`
3. `gradlew pickPg` or 'gradlew pickMSSQL`
4. `gradlew deployApp`
5. Within IntelliJ, do a Gradle Refresh in the Gradle Window
6. To start Tomcat within IntelliJ, select LabKeyEmbedded_Dev from the Run Configurations. (It will appear in a different section of the menu from the other LabKey configurations).
```